tox.ini

```diff
@@ -1,19 +1,17 @@
 [tox]
 # if you change the envlist, please update .travis.yml file as well
 envlist=
-  py{27,35,36}-pytest{29,30,31}
+  py{27,35,36}-pytest{31,latest}
```

Comment:
i wonder if we should add 4.0 and/or 4.1 to the matrix just for safety

Reply:
latest is already 4.1, I will add 4.0 then

Reply:
Done.